app/views/admin/users/new.html.erb

```diff
@@ -1,3 +1,5 @@
+<% content_for :title, "Create user" -%>
```

Comment:
We should keep the capitalization consistent, I think I would prefer `Create User`.